params/bamford_buzzard.R

```diff
@@ -0,0 +1,15 @@
+# !! PLEASE DO NOT EDIT THIS FILE !!
+# ** make a copy and edit that instead **
+# ________________________________________________
+# input parameters
+
+# sigma2: 0 < value < 5
+sig2 <- 3.2
+    
+# e.g. "anas_krystallinus"    
+species.name <- "bamford_buzzard"
```

Comment:
Hi @SamaraMedina we've had a few bamford buzzards, would you mind changing the `species.name` please?

Reply:
Change done, thanks.